odoo/addons/base/models/ir_cron.py

```diff
@@ -395,6 +395,16 @@ def toggle(self, model, domain):
         active = bool(self.env[model].search_count(domain))
         return self.try_write({'active': active})
 
+    @api.model
+    def _neutralize(self):
+        super()._neutralize()
+        for cron in self.search([]):
+            _logger.info('Deactivating cron `%s`', cron.cron_name)
+            res = cron.try_write({'active': False})
+            if not res:
+                _logger.info('Deactivation failed for cron `%s`', cron.cron_name)
+        assert self.search([('active', '=', True)]) == self.env['ir.cron'], "Not all cron were neutralized !"
```

Comment:
The logging should be a warning.
And the `assert` looks redundant with the `if not res` above, doesn't it?